restapi/src/test/java/io/stargate/db/datastore/AlphabeticalOrderPartitionKeyDecorator.java

```diff
@@ -43,4 +44,11 @@ public AlphabeticalOrderPartitionKeyDecorator(List<Column> allColumns) {
     //noinspection unchecked
     return (ComparableKey<T>) new ComparableKey<>(String.class, decorated);
   }
+
+  @Override
+  public ByteBuffer getComparableBytes(Row row) {
+    // TODO replace this with the relevant row's byte-comparable value when
+    // https://github.com/apache/cassandra/pull/1294 is ready
+    return ByteBuffer.wrap(new byte[] {});
```

Comment:
let's implement as same as the in the above method, just move that string bytes into the byte buffer?